### DISCUSSION TITLE
How I use ESP32-audioI2S library to output Bluetooth audio?

### DISCUSSION THREAD

**Discussion by renatoianhez** [pschatzmann/ESP32-A2DP#540] [Q&A] [answered]
Can I use [ESP32-audioI2S](https://github.com/schreibfaul1/ESP32-audioI2S) library to output A2DP Bluetooth? How?
I use this:
```
#include "Audio.h" // audio lib
#include "BluetoothA2DPSink.h" // Bluetooth lib

Audio audio;
BluetoothA2DPSink a2dp_sink(audio);
```

But give this error: no matching function for call to 'BluetoothA2DPSink::BluetoothA2DPSink(Audio&)'

**Comment by pschatzmann** [comment, marked as answer]
Since Audio does not inherit from Arduino Print you will need to write your own integration layer!
